### DISCUSSION TITLE
Networking streaming directly from Sony A7IV camera (firmware 3.01 with built-in SRT/RTMP) to MediaMTX fails

### DISCUSSION THREAD

**Discussion by Majestic7979** [bluenviron/mediamtx#3387] [Questions]
### Question

Hi,

I use a Sony A7IV and with Firmware 3.0/3.01 from Sony there is now a way to use the Sony Creator's App for Android to set up the camera to stream directly via SRT/RTMP to a server, the camera connects to WiFi and handles the encoding and transmission without being connected to another device. This is great, it cuts out the middle-man.

I am trying to ingest the video from my camera into MediaMTX but it's not working. In the logs the port 1935 is open for RTMP and 8890 is open for SRT.

In the Creator's app no matter what I enter for either SRT or RTMP the camera will not accept the settings.

If I try to stream my webcam from my laptop to MediaMTX using this URL in the latest OBS just downloaded from their official website as of May 24th 2024) it also doesn't work...

```
srt://10.0.0.111:8890?streamid=publish:mystream:user1:pass1&pkt_size=1316
```
I am not sure how to make this work, what am I doing wrong!? it also seems to expect a Stream key that I can't find any way to specify in MediaMTX.yml I would be grateful for some input please.

**Comment by aler9** [comment]
Please open an issue for each separate question (Sony Creator App, OBS), provide instructions on how to replicate the bug, attach server logs and network dumps as mentioned in the instructions.
